src/lwc/gsApplicationStatus/gsApplicationStatus.js-meta.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
-    <masterLabel>Component that displays the Trial Org and Application Status</masterLabel>
+    <masterLabel>NPSP Application status</masterLabel>
+    <description>Component that displays the Trial Org and Application Status</description>
     <apiVersion>49.0</apiVersion>
```

Comment:
Noting that there is a mix of API versions throughout the Getting Started pages. I'd prefer to see everything at the same api level. 50 is fine at this point.